odoo/tools/misc.py

```diff
@@ -1372,36 +1373,64 @@ def babel_locale_parse(lang_code):
         except:
             return babel.Locale.parse("en_US")
 
-def formatLang(env, value, digits=None, grouping=True, monetary=False, dp=False, currency_obj=False):
+def formatLang(env, value, digits=2, grouping=True, monetary=False, dp=None, currency_obj=None, rounding_method='HALF-EVEN', rounding_unit='decimals'):
     """
-        Assuming 'Account' decimal.precision=3:
-            formatLang(value) -> digits=2 (default)
-            formatLang(value, digits=4) -> digits=4
-            formatLang(value, dp='Account') -> digits=3
-            formatLang(value, digits=5, dp='Account') -> digits=5
+    This function will format a number 'value' to the appropriate format of the language used.
+
+    :param env: The environment.
+    :param float value: The value to be formatted.
+    :param int digits: The number of decimals digits.
+    :param boolean grouping: Usage of language grouping or not.
+    :param monetary (deprecated !): Usage of thousands separator or not.
```

Comment:
```suggestion
    :param bool monetary: Usage of thousands separator or not.
```
Please use the [deprecated](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated) directive :nerd_face: